README.md

```diff
@@ -34,12 +34,93 @@ Application-level tracing for Rust.
 structured, event-based diagnostic information. `tracing` is maintained by the
 Tokio project, but does _not_ require the `tokio` runtime to be used.
 
+## Getting Started
+
+To get started, configure `Cargo.toml` to include `tracing` and `tracing-subscriber` as dependencies. `tracing` is the core library, while `tracing-subscriber` is the part that implements `Subscriber`, which will record the trace. Take a look at the [Project layout](#Project layout) section.
+
+```
+[dependencies]
+tracing = "0.1.5"
+tracing-subscriber = "0.1.5"
+```
+
+Next, in our `main.rs` file, we will start by adding the libraries
+
+```
+use tracing::{debug, info, instrument};
+use tracing_subscriber::fmt;
+```
+
+For this simple example, we will create a function that prints sequence of numbers and we will instrument that cod to enable tracing. Tracing is enabled by initiating a span and the `#[instrument]` attribute, implicitly create a span for that function.
+
+```
+// using instrument to setup a span and skip to exclude args
+#[instrument(skip(_too_long_string))]
+fn count(nums: i32, _too_long_string: &str) -> Vec<i32> {
```

Comment:
What do you think about starting with a basic example of the use of `instrument`, like:
```rust
#[instrument]
fn count(nums: i32) -> Vec<i32>
```
and _then_ introducing the `skip` argument after showing the basic usage first?

Reply:
I'm, on board with this. I was thinking that I could introduce an argument to the instrument and explain it later down the road. But, yes, let's go with basic usage first.